tests/test_session.py

```diff
@@ -290,30 +335,3 @@ def test_patch(session: Session):
         m.patch('http://citrine-testing.fake/api/v1/bar/something', status_code=200, json=json_to_validate)
         response_json = session.patch_resource('bar/something', {"ignored": "true"})
         assert response_json == json_to_validate
-
-
-@mock.patch.object(Session, '_check_accounts_version')
-@mock.patch.object(Session, '_refresh_access_token')
-def test_base_url_assembly(*_):
```

Comment:
I'm not clear why we're removing this test.

Reply:
It's redundant with the tests above, and this test is written to do clever things depending on implementation.  For example, if you actually were to call `Citrine(host=None)`, you get an `InvalidURL` exception fired off from deeper in the stack, so under no circumstances would we care how it builds a URL with a null host.  But this test mocks the deeper calls, so this error is bypassed and a fatal scenario is treated as reasonable.